### PR TITLE
4.2 update

### PR DESCRIPTION
**Pull request by fbiville**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
+
+# gem "rails"
```
Comment (2hdddg) [inline Gemfile]: Copy paste leftover

File: index.md [hunk 2]
```diff
@@ -18,7 +21,7 @@ The sections below contain links to current, historic, and proposed future speci
 PackStream is a binary [presentation](https://en.wikipedia.org/wiki/Presentation_layer) format for the exchange of richly-typed data.
 It provides a syntax layer for the Bolt messaging protocol.
 
-- [**Version 1**](packstream/packstream-specification-1.md), corresponds to the first releases of the PackStream specification.
+- [**Version 1**]({{ site.baseurl }}{% link packstream/packstream-specification-1.md %}), corresponds to the first releases of the PackStream specification.
```
Comment (2hdddg) [inline index.md]: Are more links than this one affected? Isn't the base url needed for links below as well?

**Review by 2hdddg — APPROVED** [review]
Good initiative to be able to run locally!

**Review by bigmontz — APPROVED** [review]
📜

File: index.md (hunk 2 above)
Comment (fbiville) [reply index.md]: Oops, should have removed this

**Comment by fbiville** [comment]
Found a way to make links work locally 🎉 
Pushing that soon.

**Review by 2hdddg — APPROVED** [review]

**Pull request merged**